database/migrations/2023_03_13_156872_add_leitweg_id_to_clients_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+
+        Schema::table('clients', function (Blueprint $table) {
+            $table->string('leitweg_id')->default(null);
```

Comment:
We will want to reuse this column for other e invoicing standards, can we change this field to routing_id please